src/Mobs/Wolf.cpp

```diff
@@ -16,7 +17,6 @@ cWolf::cWolf(void) :
 	m_IsTame(false),
 	m_IsBegging(false),
 	m_IsAngry(false),
-	m_OwnerName(""),
```

Comment:
Why was this removed?

Reply:
my local clang said that this is redundant string initialization